Intro_markdown.Rmd

```diff
@@ -225,30 +225,40 @@ offenders <- readr::read_csv("Offenders_Chicago_Police_Dept_Main.csv")
 
+Note that this code is only suitable for csv files, so it is assumed by default that the first line of the file contains a header (header = T) and the columns are separated by a comma symbol (sep = ",").
+
+There are other commands and various packages that can be used to import datasets with other extensions (e.g. .xls) e.g. see http://www.statmethods.net/input/importingdata.html.
+
+
 ### Importing data from the Analytical Platform cloud storage (Amazon S3)
 
-#### Importing data using the `botor` package {-}
+Data that has been approved for storage on the Analytical Platform is generally stored in a data source (referred to as a 'bucket') on Amazon S3, which is the cloud storage solution used by the Analytical Platform. There are two options for packages that can be used to import data from S3: `Rs3tools` and `botor`.
 
-Data that has been approved for storage on the Analytical Platform is generally stored in a data source (referred to as a 'bucket') on Amazon S3, which is the cloud storage solution used by the Analytical Platform. To import the data for this session from the 'alpha-r-training' S3 bucket use this command:
+#### Importing data using the `Rs3tools` package {-}
+
+The `Rs3tools` package is maintained by other analysts in MoJ and has the advantage of being R-native, meaning it is simpler to install and use compared to `botor`. Here's how it can be installed:
 
 ```{r, results="hide"}
-offenders <- botor::s3_read("s3://alpha-r-training/intro-r-training/Offenders_Chicago_Police_Dept_Main.csv", read_csv)
+install.packages("moj-analytical-services/Rs3tools")
 ```
 
-Both this code and the code in the previous subsection are suitable for csv files, so it is assumed by default that the first line of the file contains a header (header = T) and the columns are separated by a comma symbol (sep = ",").
-
-There are other commands and various packages that can be used to import datasets with other extensions (e.g. .xls) e.g. see http://www.statmethods.net/input/importingdata.html.
+And here's how to use `Rs3tools` to read in the `offenders` dataset that we'll be using in this session from the 'alpha-r-training' S3 bucket:
 
-#### Importing data using the `Rs3tools` package {-}
+```{r, results="hide"}
+offenders <- Rs3tools::s3_path_to_full_df("alpha-r-training/intro-r-training/Offenders_Chicago_Police_Dept_Main.csv")
+```
 
-The `Rs3tools` package is an alternative to `botor` and is maintained by other analysts in MoJ. The main advantage is that it is R-native, meaning it doesn't have the slightly complicated dependencies that `botor` has. The disadvantage is that it isn't officially maintained, so must be maintained by the analytical community.
+#### Importing data using the `botor` package {-}
 
-The following code shows how to use `Rs3tools` to read in the `offenders` dataset from s3:
+The `botor` package provides an alternative method for reading from S3, which can be done using this command:
```

Comment:
How about:
The `botor` package provides an alternative to 'Rs3tools'. It's maintained by the wider R community and contains a larger range of functionality. Reading from S3 using 'botor' can be done using this command: